### PR TITLE
Announce the Governance Working Group

### PR DESCRIPTION
**Pull request by gnunicorn**
still finalizing the announcement post, attempt to merge by May 30th.

### CODE REVIEW

File: posts/2019-05-30-governance-wg-announcement.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+layout: post
+title: "The Governance WG is going public"
+author: The Rust Governance WG
+release: false
+---
+
+
+Hey all! Today we're happy to announce the [Governance Working Group](https://internals.rust-lang.org/t/governance-working-group-announcement/9637) is going public. We've been spending the last couple weeks finding our bearings and structuring the working group.
+
+You can find [our charter](https://github.com/rust-lang/wg-governance/blob/master/CHARTA.md) outlining our main goals and priorities in [our work repository](https://github.com/rust-lang/wg-governance). We are using the Github issues, milestones and projects to organise and track our progress. The [readme in the repository](https://github.com/rust-lang/wg-governance/) explains our working process a bit more in detail. It also states how you can talk to us (hint, via discord) and [get involved](https://github.com/rust-lang/wg-governance/#how-can-i-get-involved).
```
Comment (gnunicorn) [inline posts/2019-05-30-governance-wg-announcement.md]:
```suggestion
You can find [our charter](https://github.com/rust-lang/wg-governance/blob/master/CHARTER.md) outlining our main goals and priorities in [our work repository](https://github.com/rust-lang/wg-governance). We are using the Github issues, milestones and projects to organise and track our progress. The [readme in the repository](https://github.com/rust-lang/wg-governance/) explains our working process a bit more in detail. It also states how you can talk to us (hint, via discord) and [get involved](https://github.com/rust-lang/wg-governance/#how-can-i-get-involved).
```

**Comment by gnunicorn** [comment]
Ready for merge,

cc @skade

**Comment by skade** [comment]
r+ @alexcrichton

**Pull request merged**